### ISSUE TITLE
Needle lengths are wrong on spreadsheet

### ISSUE THREAD

**Issue by michaelwkudla** [brachify/brachify#17]
The spreadsheet numbers seem incorrect. For a 160mm needle length, it's showing weird lengths. I will dig into this. 

- [ ] Default needle 200mm
- [ ] Ensure needle length is being updated correctly
- [ ] Ensure used needle length is being calculated correctly.

**Comment by michaelwkudla** [comment]
Corrected in https://github.com/brachify/brachify/pull/18

**Comment by nsmela** [comment]
safe to close?

**Comment by michaelwkudla** [comment]
Not quite, i need to add a bit of error checking. Closed the PR for now.
